### ISSUE TITLE
Make conditions AND/OR and thus enable several conditions of same type

### ISSUE THREAD

**Issue by MrGeneration** [zammad/zammad#2185]
### Infos:

* Used Zammad version: 2.6.x
* Installation method (source, package, ..): any
* Operating system: any
* Database + version: any
* Elasticsearch version: any
* Browser + version: any
* Ticket-ID: #1031039, #1060690, #1067221, #1072409, #1079103, #1079241, #1079346, #1079468, #1082065, #1080184, #1083072, #1083556, #1083611, #1084691, #1087205, #1090351, #1099619 

### Expected behavior:

* When ever using conditions (e.g. for Overviews or Triggers) it should be possible to use OR condition (e.g. using the same type of statement more than once )

### Actual behavior:

* Zammad only allows AND-Conditions
* It's not possible to use a Statement type more than once per query

This applies at least to the following Zammad-Sections and should be done at all possible places, because the user would not expect the dialoges to be inconsistend:

- Overviews
- Trigger
- Scheduler
- SLAs
- Time Accounting

### Steps to reproduce the behavior:

The following applies to any Zammad-Section mentioned above, I'll use overviews as example

* Create a new overview, giving it a meaningfull name
* As condition, choose "OWNER is specific user AGENT A"
* press + for a new condition part
* try to choose owner, to make the condition match to AGENT B, too

=> won't work - this should be possible.

This Issue is to cover all request too 100%, thus the other issues will be closed and linked below:
#885 
#1004 

### Tasks

- [x] https://github.com/zammad/coordination-sprint-tasks/issues/10
- [x] https://github.com/zammad/coordination-sprint-tasks/issues/6

**Comment by martinvonwittich** [comment]
Using the same condition more than once would be useful even without OR operators. For example, the following condition is currently impossible due to the fact that Zammad allows me to use _Created at_ only once:

- Created at - after (absolute) - `2019-01-25` - at - `08:00`
- Created at - before (absolute) - `2019-01-28` - at - `08:00`

I had imported a lot of mails to stress test Zammad, and now I have ~20000 open tickets. I had hoped to use the Scheduler to close all those tickets (but not some test tickets that I created before and after the big import), and that's why I wanted to match tickets in this timeframe.
